src/workerd/io/tracer.c++

```diff
@@ -576,4 +576,17 @@ void WorkerTracer::setJsRpcInfo(const tracing::InvocationSpanContext& context,
   }
 }
 
+kj::Own<SpanObserver> UserObserver::newChild() {
+  return kj::refcounted<UserObserver>(kj::addRef(*submitter), spanId);
```

Comment:
Tiniest of nits: it would be nice if we could start moving more to using `kj::rc` and `kj::Rc<T>`

Reply:
Unfortunately this is not a tiny nit: Changing this to return a `kj::Rc<SpanObserver>` would require changes in many different files, and complicate code since we could no longer call map() in functions like `SpanBuilder::newChild()` since we depend on the `class Maybe<Own<T, D>>` specialization of kj::Maybe available in src/kj/memory.h. This is better done in another PR.